src/style.css

```diff
@@ -0,0 +1,481 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Slab:wght@300;700&display=swap');
+
+:root {
+  --primary-bg-color: #f5f6fa;
+  --secondary-bg-color: #fafafa;
+  --primary-font-color: #0290ff;
+  --secondary-font-color: #121212;
+  --tertiary-font-color: #4386bf;
+  --primary-border-color: #e8e8e8;
+  --secondary-border-color: #f0f0f0;
+  --font-family-primary: 'Montserrat', sans-serif;
+  --font-family-secondary: 'Roboto Slab', serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.flex {
+  display: flex;
+}
+
+.center {
+  justify-content: center;
+}
+
+.fontsty-normal {
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
+.montserrat {
+  font-family: var(--font-family-primary);
+}
+
+.roboto {
+  font-family: var(--font-family-secondary);
+}
+
+body {
+  display: flex;
+  height: 68.625rem;
+  background-color: var(--primary-bg-color);
+}
+
+.page {
+  margin: auto;
+  border-radius: 4px;
+  align-items: center;
+}
+
+.wrapper {
+  width: 100vw;
```

Comment:
- [ **OPTIONAL**] I think your app will look nicer if you remove this fixed width and ensure that the app is centred on the screen as on the template.